src/zenml/cli/utils.py

```diff
@@ -16,6 +16,9 @@
 import os
 import subprocess
 import sys
+import json
+import yaml
+import ast
```

Comment:
you shouldn't need to use an additional library like ast to evaluate the JSON/YAML contents. Just use `yaml.safe_load` and try/catch any errors that it may throw during parsing.

Reply:
I understand. Perhaps we've been too broad with the #620 issue description and included `dict` in the list of possible format this new value can take. We should have limited that to JSON/YAML. Using `dict` is not advisable because using `eval` on user-entered input is not a "safe" operation and could have damaging results. We don't use that anywhere else in ZenML for that same reason.